howtos/how-to-create-a-poll/server.js

```diff
@@ -0,0 +1,153 @@
+import express from "express";
+import fetch from "node-fetch";
+import cors from "cors";
+
+// Replace with your app ID and secret key
+const appId = "<APP_ID>";
+const secretKey = "<SECRET_KEY>";
+
+const basePath = "https://api.talkjs.com";
+
+const app = express();
+app.use(express.static('public'));
+app.use(cors());
+app.use(express.json());
+
+app.listen(3000, () => console.log("Server is up"));
+
+// EVERYTHING BELOW IS SETUP CODE FOR THIS EXAMPLE
+// You won't need any of it in your live app!
+//
+// It's just here so that you can play around with this example more easily
+// Whenever you start the server, we make sure the two example users are created
+// recreate the two conversations, and send messages from the example users
+
+async function setupConversation(i) {
+  const conversationId = `conversationExample${i}`;
+  const userId = `conversationExampleUser${i}`;
+
+  // Delete the conversation (if it exists)
+  await fetch(`${basePath}/v1/${appId}/conversations/${conversationId}`, {
+    method: "DELETE",
+    headers: {
+      Authorization: `Bearer ${secretKey}`,
+    },
+  });
+
+  // Create a new conversation
+  await fetch(`${basePath}/v1/${appId}/conversations/${conversationId}`, {
+    method: "PUT",
+    headers: {
+      "Content-Type": "application/json",
+      Authorization: `Bearer ${secretKey}`,
+    },
+    body: JSON.stringify({
+      participants: ["conversationExampleSupportAgent", userId],
+    }),
+  });
+
+  // Send a message from the user to make sure it will show up in the conversation list
+  await fetch(
+    `${basePath}/v1/${appId}/conversations/${conversationId}/messages`,
+    {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+        Authorization: `Bearer ${secretKey}`,
+      },
+      body: JSON.stringify([
+        {
+          text: "Everything is broken again!",
```

Comment:
Probably best to change this for the tutorial screenshot! (I know we used it ourselves, but I think it was relevant to a support related tutorial...)

Reply:
Done!